CSharp/Library/Microsoft.Bot.Connector.Teams.Shared/ConversationExtensions.cs

```diff
@@ -61,11 +65,162 @@ public static async Task<TeamsChannelAccount[]> GetTeamsConversationMembersAsync
         {
             using (var memberList = await conversations.GetConversationMembersWithHttpMessagesAsync(conversationId).ConfigureAwait(false))
             {
-                var members = await memberList.HandleErrorAsync<ChannelAccount[]>().ConfigureAwait(false);
+                var members = memberList.Body;
                 return members.Select(member => member.AsTeamsChannelAccount()).ToArray();
             }
         }
 
+        /// <summary>
+        /// GET Paged teams conversation members asynchronously.
+        /// </summary>
+        /// <param name="conversations">Conversation instance.</param>
+        /// <param name="conversationId">Conversation Id.</param>
+        /// <param name="pageSize">Requested Page size</param>
+        /// <param name="continuationToken">Continuation token to fetch more pages</param>
```

Comment:
I am guessing this is wrapping the BotFramework paged methods. In which case why do we need these methods at all? Can't AsTeamsChannelAccount just be called on individual instances to get TeamsChannelAccount equivalent? Instead of wrapping every single method.

Reply:
Correct, not a super crucial method, but it's easier from the point of view of dev, to just use this method, instead of trying to write their own rappers